samples/testing/classical_values/src/TestCode.qs

```diff
@@ -0,0 +1,27 @@
+import Std.Diagnostics.Fact;
+open Std.Arrays;
+open Std.Random;
```

Comment:
To do the equivalent of `open` with the new import syntax, you'll want to use glob imports:

```suggestion
import Std.Arrays.*;
import Std.Random.*;
```

Reply:
That looks more verbose, is there an advantage to doing that instead of open?